bin/custom/questions/src/de/hybris/questions/jobs/SendEmailWithQuestionsJobPerformable.java

```diff
@@ -0,0 +1,116 @@
+package de.hybris.questions.jobs;
+
+import de.hybris.platform.acceleratorservices.email.EmailService;
+import de.hybris.platform.core.model.user.CustomerModel;
+import de.hybris.platform.cronjob.enums.CronJobResult;
+import de.hybris.platform.cronjob.enums.CronJobStatus;
+import de.hybris.platform.processengine.BusinessProcessService;
+import de.hybris.platform.servicelayer.cronjob.AbstractJobPerformable;
+import de.hybris.platform.servicelayer.cronjob.PerformResult;
+import de.hybris.platform.servicelayer.model.ModelService;
+import de.hybris.platform.site.BaseSiteService;
+import de.hybris.platform.store.services.BaseStoreService;
+import de.hybris.questions.exception.ItemNotFoundException;
+import de.hybris.questions.model.QuestionModel;
+import de.hybris.questions.model.SendEmailWithQuestionsCronJobModel;
+import de.hybris.questions.model.SendQuestionsEmailProcessModel;
+import de.hybris.questions.model.TimeOfSendEmailWithQuestionsModel;
+import de.hybris.questions.service.QuestionsService;
+import de.hybris.questions.service.TimeOfSendEmailWithQuestionsService;
+import org.apache.log4j.Logger;
+
+import java.util.Date;
+import java.util.List;
+
+public class SendEmailWithQuestionsJobPerformable extends AbstractJobPerformable<SendEmailWithQuestionsCronJobModel> {
+
+    private static final Logger LOG = Logger.getLogger(SendEmailWithQuestionsJobPerformable.class);
+
+    private static final String CODE = "timeCode";
+
+    private ModelService modelService;
+    private EmailService emailService;
+    private BusinessProcessService businessProcessService;
+    private TimeOfSendEmailWithQuestionsService timeOfSendEmailWithQuestionsService;
+    private QuestionsService questionsService;
+    private BaseSiteService baseSiteService;
+    private BaseStoreService baseStoreService;
+
+    @Override
+    public PerformResult perform(SendEmailWithQuestionsCronJobModel sendEmailWithQuestionsCronJobModel) {
+        List<QuestionModel> questionModels;
+        TimeOfSendEmailWithQuestionsModel timeOfSendEmailWithQuestionsModel;
+
+        try {
+            timeOfSendEmailWithQuestionsModel = timeOfSendEmailWithQuestionsService.getTime();
```

Comment:
Every CronJob already contains its last start time in hybris. These service and itemtype are redundant